libfragmentzip.mk

```diff
@@ -0,0 +1,44 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS  += libfragmentzip
```

Comment:
Could you format these correctly?
All `=` should be vertically parallel directly above/below another one.

Reply:
thought i fixed that with [c65bcf6](https://github.com/ProcursusTeam/Procursus/pull/22/commits/c65bcf63e76168444b64e2be72d87202a969284a), but must've undone it, weird